src/pages/dtrader/_dtrader-tabs.tsx

```diff
@@ -270,8 +271,15 @@ class DtraderTabs extends React.Component {
                         muted
                         playsInline
                     >
-                        <source src="/Dtrader_GIF.mp4" type="video/mp4" />
-                        <source src="/Dtrader_GIF.webm" type="video/webm" />
+                        <ROW>
+                            <>
```

Comment:
This empty tag not needed

Reply:
If I remove the fragment I get this error: This JSX tag's 'children' prop expects a single child of type 'ReactElement<any, string | JSXElementConstructor>', but multiple children were provided.